ingest/ledgerbackend/toml.go

```diff
@@ -321,6 +324,12 @@ func (c *CaptiveCoreToml) unmarshal(data []byte, strict bool) error {
 	return nil
 }
 
+type FastHTTPServerParams struct {
```

Comment:
```suggestion
type HTTPQueryServerParams struct {
```

It seems that core is naming this server as the http query server since the toml fields all include "query" as a substring. So, I think a better name  would be HTTPQueryServerParams. If you accept this name suggestion then the comment in CaptiveCoreTomlParams  can be updated to:

```go
	// Enable the HTTP query server with these parameters
	HTTPQueryServerParams *HTTPQueryServerParams
```